Topics/User_Experience/Color_Theory.md

```diff
@@ -0,0 +1,73 @@
+# Color Theory
+
+## Introduction
+Color is a crucial aspect of any interface as it sets the tone, evokes emotions, and influences user perception. As such, the choice of color can impact usability and the overall user experience. With that said, there is a common misconception that the choice of colors for UI depends on the designer's aesthetic preferences and appeal. This article serves to debunk the misconceptions surrounding this notion and show that the decision-making process for generating and designing these visual elements has more layers to it than it seems.
+
+## Importance of Color on User Experience
+**Color psychology** is the study of how colors affect perception and behaviors, whether that be under the user's conscience or not. The moment our eyes perceive color, they connect with the brain which gives signals to release hormones responsible for the shifts in mood and emotions; research has shown that it takes only 90 seconds for people to make a subconscious judgment about a product and between 62% and 90% of that assessment is based on color alone. As such, color psychology forms the basis of how designers choose colors for a product based on what we want to convey.
+
+## Meaning of Colors and Association to Design
+Every color is associated with an emotion that it elicits. In this documentation, I will cover the 6 main colors - red, yellow, green, blue, black, and white - and their effect on the product.
```

Comment:
This is really cool, it might be good to add common examples of brands using this so the reader can make these connections, some immediate ones that come to mind are Tinder's logo being RED to evoke feelings of love and passion, and Uber using mainly black and white for a modern feel.

Reply:
That's a good point. Thank you for pointing that out!